### ISSUE TITLE
1NT-hand should not use blackwood

### ISSUE THREAD

**Issue by ThorvaldAagaard** [EdwardPiwowar/BBA#511]
682110FD14E087F7420F4D6F53BC
![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/ddffd9a3-b8ff-435f-8e2a-6ae6c0e64686)

Now after 3C south has a fine hand for a club slam if that is what North is looking for.

3D shows exactly 2C - after 3D North strangely bids 3H instead of 3N
3Ma is just a 3-card (with a stopper)

**Comment by EdwardPiwowar** [comment]
Next BBA
![image](https://github.com/EdwardPiwowar/BBA/assets/150140020/d50fceff-9bd1-45c6-8c3e-2d4014438366)

**Comment by ThorvaldAagaard** [comment]
Better, but I don't understand the 3D bid

**Comment by ThorvaldAagaard** [comment]
This is what might happen
046FA6FC5406F7950D0DD07BFCDB
![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/b101cf0c-bead-4998-bd6b-99694c294b8d)

![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/1b166d44-f0d3-494c-9f71-a78addd9db49)

**Comment by ThorvaldAagaard** [comment]
No, 3D is just wrong. Yes, it might show 5D and maximum, but to make anu sense it must be to avoid 3N and that is to advanced for this system, so after an invitational 2N, there are only the following options
1: Pass
2: 3N
3; A non-forcing bid guessing it is better than 2N

**Comment by EdwardPiwowar** [comment]
Next BBA
![image](https://github.com/user-attachments/assets/ef4b3696-d207-4909-bd81-6948641205d0)
